packages/content/src/requests/items/formField/settings/DateFormFieldSettings.ts

```diff
@@ -0,0 +1,32 @@
+import { type } from "@js-soft/ts-serval";
+import { CoreDate } from "@nmshd/core-types";
+import { FormFieldSettings, FormFieldSettingsJSON, IFormFieldSettings } from "./FormFieldSettings";
+
+export interface DateFormFieldSettingsJSON extends FormFieldSettingsJSON {
+    "@type": "DateFormFieldSettings";
+}
+
+export interface IDateFormFieldSettings extends IFormFieldSettings {}
+
+@type("DateFormFieldSettings")
+export class DateFormFieldSettings extends FormFieldSettings implements IDateFormFieldSettings {
+    public canCreate(): string | undefined {
+        return;
+    }
+
+    public canAccept(response: string | number | boolean | string[]): string | undefined {
+        if (!(typeof response === "string" && CoreDate.from(response).dateTime.isValid)) {
```

Comment:
This would be way easier to read.
```suggestion
        if (typeof response !== "string" || !CoreDate.from(response).dateTime.isValid) {
```